basiclti/basiclti-common/src/java/org/sakaiproject/basiclti/util/SakaiBLTIUtil.java

```diff
@@ -935,10 +935,83 @@ public static void addGlobalData(Site site, Properties props, Properties custom,
 				M_log.debug("Launching with SHA256 Signing");
 			}
 		}
+		
+		//substitute resources params and add the rest
+		if(custom.getProperty(BasicLTIConstants.RESOURCE_LINK_ID) != null){//TODO if allowReviewService prop y future prop for TII
+			addTIIproperties(ltiProps, custom);
```

Comment:
This project should be solely focused on LTI without code specific to any individual LTI integrations; addTIIproperties should be moved to the Turnitin content-review implementation